src/test/java/test/ReflectionHelperTest.java

```diff
@@ -0,0 +1,64 @@
+package test;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.collections.Lists;
+import org.testng.internal.reflect.ReflectionHelper;
+import test.github765.DuplicateCallsSample;
+
+import java.lang.reflect.Method;
+import java.util.List;
+
+public class ReflectionHelperTest {
+
+    @Test
+    public void testMethodCount() {
```

Comment:
@juherr - a method from a generics class gets regarded as a bridged method only when there is inheritance involved. So no point in adding a test case for bridge methods without inheritance.